internal/handler/wallet/sign.go

```diff
@@ -83,7 +134,7 @@ func (s *walletSign) Handle(params operations.SignParams) middleware.Responder {
 
 	signature, err := wlt.Sign(op)
 	if err != nil {
-		return operations.NewSignInternalServerError().WithPayload(
+		return operations.NewSignBadRequest().WithPayload(
```

Comment:
no, this is really a internal server error if we can't sign, not an error due to user inputs.